### PR TITLE
Add GeoJSON support

### PR DESCRIPTION
**Pull request by VinceBT**
See #8

### CODE REVIEW

**Review by dcousens — APPROVED** [review]

File: index.d.ts [hunk 1]
```diff
@@ -13,29 +13,19 @@ interface LatLon {
...
- */
-declare function haversineDistance(a: LatitudeLongitude, b: LatitudeLongitude): number;
+interface GeoJSONPoint extends Array<number|number>{0:number; 1:number}
 
-/**
- * Return the Haversine distance in meters
- * @param a - first location
- * @param b - second location
- */
-declare function haversineDistance(a: LatLng, b: LatLng): number;
```
Comment (dcousens) [inline index.d.ts:28]: Although these were conceptually cleaner,  they don't reflect the implementation.  :+1:

Comment (VinceBT) [reply index.d.ts:28]: That's why I cleaned up a bit 😄👍
Thank you for the merge

**Comment by VBT-BB** [comment]
Are you planning on a new release ? Thanks

**Comment by VinceBT** [comment]
Are you planning on a new release ? Thanks

**Comment by dcousens** [comment]
Released as `v1.2.1`
(thanks for reminder)

**Pull request merged**